modular_zubbers/code/modules/clothing/outfits/ert.dm

```diff
@@ -0,0 +1,17 @@
+/datum/outfit/centcom/ert/commander/alert
+	l_hand = /obj/item/gun/energy/modular_laser_rifle
+
+/datum/outfit/centcom/ert/security/alert
+	l_hand = /obj/item/gun/energy/modular_laser_rifle
+	r_hand = /obj/item/gun/ballistic/automatic/smart_machine_gun
```

Comment:
these are meant to be used by the mod module, You shouldn't add it to that

Reply:
what

Reply:
the smartgun